src/util/store/repo.ts

```diff
@@ -28,21 +19,13 @@ export const checkRepoExists = (appState: AppState) => async (
 
 export const createRepoDocument = (appState: AppState) => async (
   repoId: string,
-  data: Pick<StoreRepo, 'timestamp' | 'lastGitHash' | 'lastGitTime'> &
-    Partial<Pick<StoreRepo, 'paths' | 'deleted'>>
+  data: Pick<StoreRepo, 'timestamp' | 'lastGitHash' | 'lastGitTime'>
```

Comment:
Are we still storing this? Then maybe it shouldn't be in `old-types.ts`. Also, it looks like `old-types.ts` never gets flushed out in this changeset.

Reply:
I removed old-types.ts, but we're still storing the lastGitHash and lastGitTime. It may not be needed, but I feel it is useful exposition into our old system during migration.